src/Common/src/Interop/Windows/winhttp/Interop.winhttp.cs

```diff
@@ -257,11 +238,6 @@ internal partial class WinHttp
             uint notificationFlags,
             IntPtr reserved);
 
-        [DllImport(Interop.Libraries.WinHttp, CharSet = CharSet.Unicode, SetLastError = true)]
-        public static extern SafeWinHttpHandleWithCallback WinHttpWebSocketCompleteUpgrade(
-            SafeWinHttpHandle requestHandle,
-            IntPtr context);
-
         [DllImport(Interop.Libraries.WinHttp, CharSet = CharSet.Unicode, SetLastError = false)]
         public static extern uint WinHttpWebSocketSend(
```

Comment:
You also delete all the WinHttpWebSocket* DLL imports as well since those were only used by ClientWebSocket implementation when it used WinHTTP.

Reply:
done!

thanks for pointing to that code